src/app/containers/App/App.test.jsx

```diff
@@ -28,14 +28,20 @@ describe('App', () => {
     <h1>{initialData.pageData}</h1>,
   );
 
+  const bbcOrigin = 'https://www.bbc.co.uk';
+  const bbcCountry = 'GB';
+
   beforeAll(() => {
     wrapper = mount(
       <App
         location={{ pathname: 'pathnameOne' }}
         routes={[]}
         initialData={initialData}
-        bbcOrigin="https://www.bbc.co.uk"
         history={history}
+        headers={{
```

Comment:
Can the app access the headers from the initial data prop? This saves us sending extra props and from adding more prop type declarations to the component.

Reply:
Done!